imgee/templates/delete_label.html.jinja2

```diff
@@ -10,8 +10,8 @@
   <div class="alert alert--error">
     <p class="mui--text-title">{{ faicon(icon='trash-alt', icon_size='subhead', baseline=true) }} Confirm delete</p>
     <hr/>
-    <p class="mui--text-subhead">Are you sure you want to delete the label "{{ label.title }}"?<br/></p>
-    <p>{{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>Info: Even if you delete it, the files associated with the label will NOT be deleted.</strong></p>
+    <p class="mui--text-subhead">{% trans label=label.title %}Are you sure you want to delete the label "{{ label }}{% endtrans %}"?<br/></p>
+    <p>{{ faicon(icon='info-circle', baseline=true) }}&nbsp;<strong>{% trans %}Info: Even if you delete it, the files associated with the label will NOT be deleted.{% endtrans %}</strong></p>
```

Comment:
Change to: Removing this label will dissociate it from images, but will not remove those images.